examples/example_precompiles_vm/example_precompiles_vm.cpp

```diff
@@ -75,32 +75,14 @@ static evmc_result execute(evmc_vm* /*unused*/,
     switch (id)
     {
     case 0x0001:  // ECDSARECOVER
-        return not_implemented();
```

Comment:
Why do you execute identity now for addresses 0x1-0x3 ?

Reply:
clang-tidy is complaining that many switch cases have the same code.

Reply:
Should be
```
    case 0x0001:  // ECDSARECOVER
    case 0x0002:  // SHA256
    case 0x0003:  // RIPEMD160
        return not_implemented();
    case 0x0004:  // Identity
        return execute_identity(msg);
```